xtext/editorserver/src/controllers/XtextController.js

```diff
@@ -66,14 +48,44 @@ class XtextController {
         try {
             const editorId = req.params.editorId;
             const filePath = config.deployFileLocation + "/" + editorId;
+            const buildLogPath= `${config.buildFileLocation}/${editorId}/build.log`;
+            const buildErrorLogPath= `${config.buildFileLocation}/${editorId}/build.err`;
+            const buildStatusPath= `${config.buildFileLocation}/${editorId}/build.res`;
             const editorDeployed = fs.existsSync(filePath);
+            
+            let buildLog;
+            let errorLog;
+            let buildStatus;
+
+            // Read the build log
+            try {
+                buildLog = fs.readFileSync(buildLogPath, 'utf8');
+            } catch (err) {
+                console.log("Error reading build log: " + buildLogPath);
+                console.log(err);
+            }
+
+            // Read the error log
+            try {
+                errorLog = fs.readFileSync(buildErrorLogPath, 'utf8');
+            } catch (err) {
+                console.log("Error reading build error log: " + buildErrorLogPath);
+                console.log(err);
+            }
+
+            // Read the build status
+            try {
+                buildStatus = Number( fs.readFileSync(buildStatusPath, 'utf8') );
+            } catch (err) {
+                console.log("Error reading build status: " + buildStatusPath);
+                console.log(err);
+            }
 
             let response = {};
             response.editorReady = editorDeployed;
+            response.output = buildLog;
```

Comment:
Can we get the error output to the console, too? Xtext puts errors with the model onto the error stream and thus they currently aren't sent to students (see #39)...

Reply:
Yes definitely, will include in #39 next.

Reply:
BTW, doing this here would be the wrong place as it would lose the interleaving of messages on the two streams. Instead, I would suggest doing it via the redirects in `build.sh`.

Reply:
Ok in that case I'll include it here. It would be nice to have a separate list of only the errors for the platform response's error field but less important than providing full logs to learners.

Reply:
Stderr  now included in logs in e3f041d1ea2e3bc850a54aafa196e6b2da8aa3e5